src/hooks/Circles/useJoinCircles.tsx

```diff
@@ -11,12 +11,12 @@ export function useJoinCircles() {
   const { httpClient } = useHttpClient();
 
   return useQuery(
-    [`/v1/life-research/projects/${activeProject?.id}/app-config/circles`],
+    [`/v1/life-research/projects/${activeProject.id}/app-config/circles`],
```

Comment:
Strange that this wasn't erroring out before. I'm guessing that the repo doesn't have that strict template rule set in eslint?